schemas/NotificationSignature.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "NotificationSignature",
+  "description": "The signature associated with a notification",
+  "type": "object",
+  "properties": {
+    "algorithm": {
+      "description": "Signature algorithm ID.",
+      "type": "string",
+      "enum": [
+        "PS256",
+        "ES256"
+      ]
+    },
+    "publicKey": {
+      "description": "Public key object.",
+      "$ref": "JSONSigningPublicKey.json"
+    },
+    "value": {
+      "description": "The signature data.",
+      "type": "string"
```

Comment:
I couldn't find a good regex for the signature value. It should be base64, according to the [schema](https://cyberphone.github.io/openkeystore/resources/docs/jcs.html#signature), but it doesn't validate against our Base64.json, because it contains `_` and `-` characters. Suggestions are welcome.